prettier.config.ts

```diff
@@ -1,6 +1,6 @@
-import { Options } from 'prettier'
+import * as prettier from 'prettier'
 
-const options: Options = {
+const options: prettier.Options = {
```

Comment:
The old way (`const options: Options`) stutters and it's also unclear what `Options` means here and where it comes from.

The new way (`const options: prettier.Options`) makes clear that these are options for Prettier.